### ISSUE TITLE
[FAIL] Finch Core E2E Tests display detailed information on one or more images [It] should not display information if image doesn't exist

### ISSUE THREAD

**Issue by austinvazquez** [runfinch/common-tests#205]

**Comment by austinvazquez** [comment]
Test failing for:

```
2024-12-26T18:36:18.1979170Z [38;5;9m• [FAILED] [0.995 seconds][0m
2024-12-26T18:36:18.1980020Z [0mFinch Core E2E Tests [38;5;243mdisplay detailed information on one or more images [38;5;9m[1m[It] should not display information if image doesn't exist[0m
2024-12-26T18:36:18.1980920Z [38;5;243m/Users/ec2-user/go/pkg/mod/github.com/austinvazquez/common-tests@v0.0.0-20241226182914-94aa2dc6c049/tests/image_inspect.go:48[0m
2024-12-26T18:36:18.1981330Z 
2024-12-26T18:36:18.1981430Z   [38;5;9m[FAILED] Expected
2024-12-26T18:36:18.1981620Z       <int>: 0
2024-12-26T18:36:18.1981770Z   not to match exit code:
2024-12-26T18:36:18.1981980Z       <int>: 0[0m
2024-12-26T18:36:18.1982740Z   [38;5;9mIn [1m[It][0m[38;5;9m at: [1m/Users/ec2-user/go/pkg/mod/github.com/austinvazquez/common-tests@v0.0.0-20241226182914-94aa2dc6c049/command/command.go:124[0m [38;5;243m@ 12/26/24 18:36:17.782[0m
```

**Comment by austinvazquez** [comment]
In nerdctl 2.0, image inspect on non-existent image is now exiting exit code 0.

**Comment by austinvazquez** [comment]
I believe this is an unintended upstream regression in nerdctl. I am putting together a patch to propose to resolve.

Draft is https://github.com/austinvazquez/nerdctl/pull/19

**Comment by austinvazquez** [comment]
Opened https://github.com/containerd/nerdctl/pull/3792 to resolve.

**Comment by austinvazquez** [comment]
Fix has been merged in nerdctl and will be made available in v2.0.3 tag.

I have packaged the commit in a self generated archive and tested in https://github.com/runfinch/finch-core/pull/472.
